src/App.tsx

```diff
@@ -1,61 +1,127 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. Consider moving the import statements for `usersFromServer` and `todosFromServer` closer to where they are being used within the `App` component to emphasize that they are only being used here, and to prevent them from being imported unnecessarily in other components.